04_set_up_the_folio_users_app.md

```diff
@@ -148,7 +149,7 @@ $ mvn install
 ```
 
 ### Register and Deploy the Users app Okapi Module
-The Git repository for Okapi Module has a [Module Descriptor](https://github.com/folio-org/mod-users/blob/master/ModuleDescriptor.json) and a [Deployment Descriptor](https://github.com/folio-org/mod-users/blob/master/DeploymentDescriptor.json) that can be used to register and deploy the Users app Okapi Module.
+The Git repository for the Users app Okapi Module has a [Module Descriptor](https://github.com/folio-org/mod-users/blob/master/ModuleDescriptor.json) that can be used to register the Users app Okapi Module.
```

Comment:
Just to document - the extra pairs cause Okapi to attempt to deploy on port 9131. If you follow lesson 3 before you do lesson 4, you end up with okapi-test-auth-module running on 9131, so the deployment of mod-users fails.